### ISSUE TITLE
`npm outdated` reports mismatch between 3.1.0 & 3.2.0

### ISSUE THREAD

**Issue by fboes** [dlmanning/gulp-sass#669]
When I run `npm oudated` I get the following message:

```
Package    Current  Wanted  Latest  Location
gulp-sass    3.1.0   3.2.0   3.1.0  blogophon
```

This also happens from a fresh install, as you can see in https://travis-ci.org/fboes/blogophon/builds/352715853

If I update _my_ `package.json` to `gulp-sass@3.2.0` and run `npm install` it gets even stranger:

```
Package    Current  Wanted  Latest  Location
gulp-sass    3.2.0   3.2.0   3.1.0  blogophon
```

Sadly `npm outdated` is part of my [deployment testing and makes my buid fail](https://travis-ci.org/fboes/blogophon/builds/352715853). 😢 

Possible cause
--------------

This may have something to do with `gulp-sass` having version `3.2.0` in its `package.json`, but only `3.1.0` on https://www.npmjs.com/package/gulp-sass

Addtional info
---------------

```
$ node --version
v9.8.0
$ npm --version
5.6.0
$ nvm --version
0.33.8
```

**Comment by xzyfer** [comment]
This is because we promoted 3.1.0 to latest after a bug was found in latest
node-sass. You must've installed gulp-sass 3.2.0 in the short period it was
latest. You can expect 3.2.1 to be released in the next 24hrs

**Comment by xzyfer** [comment]
I suggest explicitly installing gulp-sass@3.1.0 and clearing your npm cache
to help outdated pick up the changes.

**Comment by fboes** [comment]
I tried the best I can manage to clear my npm cache, but even from a fresh install the error persists:

```
$ npm install gulp-sass --save
...
+ gulp-sass@3.1.0
added 238 packages from 173 contributors in 19.561s

$ npm outdated
Package    Current  Wanted  Latest  Location
gulp-sass    3.1.0   3.2.0   3.1.0
```

Even stranger: A [new build on Travis CI also fails](https://travis-ci.org/fboes/blogophon/builds/352945513) while checking `npm outdated` - I was under the impression that Travis CI starts from scratch every time.

I someone out there willing to reproduce above testcase (`npm install gulp-sass --save && npm outdated`) on another machine? Maybe it is only on Travis CI, my local computer and the machine I deploy to?

**Comment by eseliger** [comment]
@fboes happening for my local machine and our CI env as well

**Comment by devwax** [comment]
Same here. I was getting an infinite loop with my watch task in gulp. I downgraded to 3.1.0 and it solved the problem.

When I ran `npm i gulp-sass -D` it installed 3.1.0 and the same weird outdated mismatch problem. I'm not using CI or anything, just local. `npm 5.7.1` `node 8.6.0` osx 10.10.5

**Comment by xzyfer** [comment]
Fixed in 3.2.1
